Sdk/Speckle.Connectors.Logging/Internal/LogBuilder.cs

```diff
@@ -18,16 +15,9 @@ public static LoggerProvider Initialize(
     ResourceBuilder resourceBuilder
   )
   {
-    var fileVersionInfo = GetFileVersionInfo();
     var serilogLogConfiguration = new LoggerConfiguration()
       .MinimumLevel.Is(SpeckleLogLevelUtility.GetLevel(speckleLogging?.MinimumLevel ?? SpeckleLogLevel.Warning))
       .Enrich.FromLogContext()
-      .Enrich.WithProperty("version", fileVersionInfo.FileVersion)
-      .Enrich.WithProperty("productVersion", connectorVersion)
-      .Enrich.WithProperty("hostOs", DetermineHostOsSlug())
```

Comment:
hmm, why don't we?

Reply:
this is now part of tags and will attach to all traces and logs as part of Otel